win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1577,16 +1599,36 @@ IntGdiAddFontResourceEx(PUNICODE_STRING FileName, DWORD Characteristics,
     PVOID Buffer = NULL;
     IO_STATUS_BLOCK Iosb;
     PVOID SectionObject;
-    SIZE_T ViewSize = 0;
+    SIZE_T ViewSize = 0, Length;
     LARGE_INTEGER SectionSize;
     OBJECT_ATTRIBUTES ObjectAttributes;
     GDI_LOAD_FONT LoadFont;
     INT FontCount;
     HANDLE KeyHandle;
+    UNICODE_STRING PathName;
+    WCHAR szPath[MAX_PATH];
     static const UNICODE_STRING TrueTypePostfix = RTL_CONSTANT_STRING(L" (TrueType)");
 
+    /* Build PathName */
```

Comment:
It would be nice to have a comment stating why we have to replace NtSystemRoot with `"\\SystemRoot"`, since in most cases the two should be equivalent.

Reply:
`NtSystemRoot` is `C:\ReactOS`, that is inaccessible from kernel, likely.